assets/javascripts/modules/enhancedTables.js

```diff
@@ -1,5 +1,5 @@
 require('jquery');
-require('datatables');
+require('datatables')();
```

Comment:
can you put a comment in here with a link to the issue - documentation as to why this has to be done this way